internal/disk/lvm.go

```diff
@@ -128,3 +132,13 @@ func (lv *LVMLogicalVolume) EnsureSize(s uint64) bool {
 	}
 	return false
 }
+
+// lvname returns a name for a logical volume based on the mountpoint.
+func lvname(path string) string {
+	if path == "/" {
+		return "rootlv"
+	}
+
+	path = strings.TrimLeft(path, "/")
+	return strings.ReplaceAll(path, "/", "_") + "lv"
```

Comment:
`/` and `/root` will end up with the same volume name. As will `/home/foo_bar` and `/home/foo/bar`.

Reply:
Yeah, I think we would need to solve this at a level higher and attach a suffix if the name already exists. `/` is not allowed in the name and any substation we do will end up with the same problem, no?